### ISSUE TITLE
Improve the widgets

### ISSUE THREAD

**Issue by tiotrom** [liberapay/liberapay.com#1274]
Would be fantastic to have it like a progress bar with the amount we want to raise and allow people to choose the amount from our website, then complete the process on Liberapay website. Would this be possible or something you may want to do in the future? Thanks!

**Comment by mattbk** [comment]
@tiotrom, have you looked at the "goal progress" badge at https://liberapay.com/about/me/widgets/? That should cover the first part of your question.

**Comment by tiotrom** [comment]
@mattbk Yes those are really cool! I was thinking if there could be a way for people to basically choose the amount they want to donate from such a widget.
